packages/template-retail-react-app/app/commerce-api/hooks/useCustomer.js

```diff
@@ -39,6 +41,16 @@ export default function useCustomer() {
                 return customer?.authType === AuthTypes.GUEST
             },
 
+            /**
+             * Returns if this customer is newly registered.
+             */
+            get isNewlyRegistered() {
+                if (!customer || customer.authType !== 'registered') return false
+                const lastLoginTimeDate = new Date(Date.parse(customer.lastLoginTime))
+                const creationDate = new Date(Date.parse(customer.creationDate))
```

Comment:
```suggestion
                const lastLoginTimeDate = Date.parse(customer.lastLoginTime)
                const creationDate = Date.parse(customer.creationDate)
```
Don't convert from number to Date if all you're doing is calculating a number. 😉

Reply:
True. I'll fix this outside of this suggestion as my OCD is telling me to rename those variables as they would no longer be Date but rather TimeStamps.